lib/use_case/health_check.rb

```diff
@@ -1,19 +1,22 @@
 module UseCase
   class HealthCheck
-    def initialize(route53_gateway: Gateways::Route53.new)
+    def initialize(route53_gateway: Gateway::Aws::Route53.new, delayer: Gateway::Delayer.new(wait_time: 5))
       @route53_gateway = route53_gateway
+      @delayer = delayer
     end
 
     def healthy?
       health_checks.all? do |health_check|
         return true if noop_parent_health_check?(health_check)
+
+        delayer.delay(wait_time: 5)
```

Comment:
Given the issue was to do with bursting within the space of a second, 5 seconds feels like a long delay.

Could this be changed to `0.5` or `1`?

Reply:
Yeah 5s was the recommendation from AWS support.  But can see that being a lifetime to complete.  Shall go for a lower value first to see what it does.

Reply:
Let's leave it with 5s then.

I did a bit of digging on the AWS Client, and it's already using exponential backoff when API Limits kick in, so don't think this is related to bursting in a second.